apps/aecore/src/aec_governance.erl

```diff
@@ -90,11 +90,14 @@ expected_block_mine_rate() ->
     aeu_env:user_config_or_env([<<"mining">>, <<"expected_mine_rate">>],
                                aecore, expected_mine_rate, ?EXPECTED_BLOCK_MINE_RATE).
 
+block_mine_reward(0) ->
```

Comment:
`0` -> `?GENESIS_HEIGHT`?

Reply:
I guess this would make sense, but this code is pretty much hard wired to having genesis at 0, so I think it is better to keep it explicit.